com/truckmuncher/api/trucks.proto

```diff
@@ -77,6 +85,19 @@ message ActiveTrucksResponse {
          * Value in the range [-180, 180]
          */
         required double longitude = 3;
+        /** 
+         * Indicates whether the truck's location is verified or not.
+         */
+        optional bool verified = 4;
+        /**
+         * Indicates the current upvote/downvote score.
+         */
+        optional int score = 5;
+        /**
+         * Signifies the user's vote for the truck. Positive numbers are upvotes, negatives are downvotes, anything else 
+         * (null or 0) means that the user has not provided a vote.
+         */
+        optional bool userVote = 6;
```

Comment:
I thought that optional would mean that it could be null. I don't know how this works because I don't use the protos.